{
    "issue": {
        "title": "Handling exceptions during execution of std::wstring_convert",
        "body": "***Note**: This issue was originally created as [ARROW-742](https://issues.apache.org/jira/browse/ARROW-742). Please see the [migration documentation](https://gist.github.com/toddfarmer/12aa88361532d21902818a6044fda4c3) for further details.*\n\n### Original Issue Description:\nstd::wstring_convert can throw exceptions on errors, but they are not handled at the moment.",
        "created_at": "2017-03-30T19:02:44.000Z",
        "updated_at": "2017-07-09T16:40:06.000Z",
        "labels": [
            "Migrated from Jira",
            "Component: C++",
            "Type: enhancement"
        ],
        "closed": true,
        "closed_at": "2017-06-13T01:13:32.000Z"
    },
    "comments": [
        {
            "created_at": "2017-06-13T01:13:32.045Z",
            "body": "***Note**: [Comment](https://issues.apache.org/jira/browse/ARROW-742?focusedCommentId=16047306) by Wes McKinney (wesm):*\nIssue resolved by pull request 749\n<https://github.com/apache/arrow/pull/749>"
        }
    ]
}